tests/conftest.py

```diff
@@ -112,7 +222,13 @@ def cleanup():
 
 @pytest.fixture(scope="session")
 def valid_image_rofs_with_mender_conf(request):
-    valid_image = "mender-image-full-cmdline-rofs-%s.ext4" % machine_name
+
+    dirname = _extract_fs_from_image(
```

Comment:
Suggestion: All these `image` fixtures share pretty much the exact same pattern:

dirname = ...

return dirname + filename

Do you think you could find a way of sharing that functionality here somehow? :)